src/specific/s_clock.c

```diff
@@ -1,50 +1,44 @@
 #include "specific/s_clock.h"
 
-#include "config.h"
-#include "global/vars.h"
+#include "global/const.h"
 
-#include <windows.h>
+#include <SDL2/SDL_stdinc.h>
+#include <SDL2/SDL_timer.h>
 
-static LONGLONG m_Ticks = 0;
+static Uint64 m_Ticks = 0;
```

Comment:
Why not `uint64_t`?

Reply:
SDL is declared to return a SDL.h Uint64 not stdint.h uint64_t.